docs/customization/model.rst

```diff
@@ -230,7 +236,9 @@ in our case it will be the ``SyliusShippingBundle`` -> ``sylius_shipping``.
         resources:
             shipping_method:
                 classes:
-                    model: App\Entity\ShippingMethod
+                    model: App\Entity\Shipping\ShippingMethod
```

Comment:
We are missing the interface config I suppose?

Reply:
And here